### PR TITLE
Fixes #13126: Generate redirection info to latest stable doc

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/13126

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,9 +36,15 @@ $(SITES): prepare rudder-theme/build/ui-bundle.zip
 	antora --ui-bundle-url ./rudder-theme/build/ui-bundle.zip $@.yml
 
 %.tar.gz: %
+	mkdir -p build/archives
 	cd build && tar -cvzf $@ $<
+	mv build/$@ build/archives/
 	rm -rf build/$<
 
+build/sites/redirect-current.conf:
```
Comment (peckpeck) [inline Makefile]: -sites ?

**Comment by amousset** [comment]
Having a separate folder for archives allow easy publication : `rsync --delete` for sites, `rsync` for archives (to keep unmaintained versions). This also allows exposing a clean archive index.

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Pull request merged**